dkc/core/tests/test_folder_rest.py

```diff
@@ -62,7 +62,8 @@ def test_folder_rest_update(api_client, folder):
 def test_folder_rest_update_parent_disallowed(api_client, folder, folder_factory):
     other_root = folder_factory()
     child = folder_factory(parent=folder)
-    api_client.patch(f'/api/v2/folders/{child.id}', data={'parent': other_root.id})
+    resp = api_client.patch(f'/api/v2/folders/{child.id}', data={'parent': other_root.id})
+    assert resp.status_code == 200
```

Comment:
I would expect that to be an error.  Is 200 the behavior dictated by DRF?

Reply:
It is indeed DRF's behavior when trying to patch a non-editable field. It ignores it and continues on. I also would have hoped for a 400 here.

Reply:
Apparently this is the default behavior of DRF when given an unknown field.  I wonder if making parent a [read_only_field](https://www.django-rest-framework.org/api-guide/serializers/#specifying-read-only-fields) would raise an error?

Reply:
I guess it doesn't... oh well, I guess this is good enough.